assets/css/main.css

```diff
@@ -477,6 +477,224 @@ ul {
+  /* ========== Global ========== */
+  /* Color of the background */
+  --global-background-color: #fff;
+  /* Color of the text */
+  --global-font-color: #161209;
+  /* Color of the secondary text */
+  --global-font-secondary-color: #4d4d4d;
+  /* Color of the link */
+  --global-link-color: #161209;
+  /* Color of the hover link */
+  --global-link-hover-color: #2d809a;
+  /* Color of the border */
+  --global-border-color: #f0f0f0;
+  /* ========== Global ========== */
+  /* ========== Selection ========== */
+  /* Color of the selected text */
+  --selection-color: rgba(53, 166, 247, 0.25);
+  /* ========== Selection ========== */
+  /* ========== Header ========== */
+  /* Color of the header background */
+  --header-background-color: #f8f8f8;
+  /* Color of the hover header item */
+  --header-hover-color: #161209;
+  --header-delimiter: unset;
+  /* Color of the search background */
+  --search-background-color: #e9e9e9;
+  --search-empty-font-color: unset;
+  /* ========== Header ========== */
+  /* ========== Single Content ========== */
+  /* Font weight of the single H2 - H6 */
+  --single-h-font-weight: bold;
+  /* Color of the single content strong/b */
+  --single-content-strong-color: unset;
+  /* Color of the single link */
+  --single-link-color: #2d809a;
+  /* Color of the hover single link */
+  --single-link-hover-color: #ef3982;
+  /* Color of the table background */
+  --table-background-color: #fff;
+  /* Color of the table thead */
+  --table-thead-color: #ededed;
+  --table-thead-color-darken: darken(#ededed, 2%);
+  /* Color of the blockquote */
+  --blockquote-color: #6bd6fd;
+  --blockquote-bg-color: rgba(107, 214, 253, 0.2);
+  /* ========== Single Content ========== */
+  /* ========== Pagination ========== */
+  /* Color of the link in pagination */
+  --pagination-link-color: #2d809a;
+  /* Color of the hover link in pagination */
+  --pagination-link-hover-color: #000;
+  /* ========== Pagination ========== */
+  /* ========== Friend Link ========== */
+  /* Color of friend link background */
+  --friend-link-background-color: #f8f8f8;
+  /* Color of friend link text */
+  --friend-link-color: #2d809a;
+  /* Color of hover friend link text */
+  --friend-link-hover-color: #ef3982;
+  /* ========== Friend Link ========== */
+  /* ========== Sponsor Button ========== */
+  /* Sponsor button background color */
+  --sponsor-button-background-color: #f0f0f0;
+  /* Sponsor button background color while in the :hover state */
+  --sponsor-button-hover-background-color: #f0f0f0;
+  /* ========== Sponsor Button ========== */
+  /* ========== Code ========== */
+  /* Color of the code */
+  --code-color: #e74c3c;
+  /* Color of the code background */
+  --code-background-color: #f5f5f5;
+  --code-background-color-darken-3: #ededed;
+  --code-background-color-darken-5: #e8e8e8;
+  --code-background-color-darken-6: #e6e6e6;
+  --code-background-color-darken-8: #e1e1e1;
+  --code-background-color-darken-10: gainsboro;
```

Comment:
Why do we need to hard code all the darken colors here? 

Can something like this work?
```css
.style {
  --color: #000;
  color: darken(var(--color), 3%);
}
```

Reply:
darken is a scss function . `color: darken(var(--color), 3%);`  

> ERROR TOCSS: failed to transform "/css/style.scss" (text/x-scss): "/root/DoIt/assets/css/_variables.scss:107:34": argument `$color` of `darken($color, $amount)` must be a color
Built in 890 ms
Error: error building site: TOCSS: failed to transform "/css/style.scss" (text/x-scss): "/root/DoIt/assets/css/_variables.scss:107:34": argument `$color` of `darken($color, $amount)` must be a color

Reply:
Mabey we should use hsl
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl

Reply:
We are not using darken-8 and darken-10 anywhere right? I can look into each individual usage point and pick an explicit color for each of them.

Reply:
https://github.com/HEIGE-PCloud/DoIt/blob/a3970ec1af30d027102cd8889701877390d5d8ba/assets/css/_partial/_single/_code.scss#L79
https://github.com/HEIGE-PCloud/DoIt/blob/a3970ec1af30d027102cd8889701877390d5d8ba/assets/css/_partial/_single/_code.scss#L130

so /root/DoIt/assets/css/tailwind.css L129 
```css
--code-background-color-code-header: var(--code-background-color-darken-8);
--code-background-color-code-hl: gainsboro; 
```
should be 

```css
--code-background-color-code-hl: var(--code-background-color-darken-10);
````

It is a err . And dark black theme don't use them ( 8 and 10 ).